components/ProgressCard.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+import '../scss/progressCard.scss'
+
+type Props = {
+  progressCount: number
+}
+
+const ProgressCard: React.FC<Props> = ({ progressCount }) => {
+  return (
+    <div className="progress-card__container card shadow-sm mt-3">
+      <div className="card-body">
+        <div className="progress-card__progress-bar">
+          <span className="d-block font-weight-bold pt-2">
+            {progressCount}%
+          </span>
+        </div>
+        <div className="progress-card__title mt-3">{`You're off to a great start!`}</div>
```

Comment:
is it possible to us `h1`, `h2`, `h3`, `h3`, etc.?  `div`'s are bad for accessibility.